src/main/java/il/org/osm/israelhiking/PlanetSearchProfile.java

```diff
@@ -745,6 +745,11 @@ private void setIconColorCategory(PointDocument pointDocument, WithTags feature)
                 pointDocument.poiIcon = "icon-star";
                 pointDocument.poiCategory = "Other";
                 return;
+          case "alpine_hut":
+                pointDocument.poiIconColor = "#734a08";
+                pointDocument.poiIcon = "icon-home";
```

Comment:
Are we sure about this?
Not that I have a better idea which icon to use...

Reply:
Also, there's some special handling for icon-home here:
https://github.com/IsraelHikingMap/planet-search/blob/a767d9e8baf28d1a15abea21fef6939b5077bfe8/src/main/java/il/org/osm/israelhiking/PlanetSearchProfile.java#L386